aiida/cmdline/commands/cmd_node.py

```diff
@@ -35,17 +35,27 @@ def verdi_node_repo():
 
 @verdi_node_repo.command('cat')
 @arguments.NODE()
-@click.argument('relative_path', type=str)
+@click.argument('relative_path', type=str, required=False)
 @with_dbenv()
 def repo_cat(node, relative_path):
-    """Output the content of a file in the node repository folder."""
+    """Output the content of a file in the node repository folder.
+
+    For ``SinglefileData`` nodes, the `RELATIVE_PATH` does not have to be specified as it is determined automatically.
+    """
     import errno
-    from shutil import copyfileobj
     import sys
 
+    from aiida.orm import SinglefileData
+
+    if not relative_path:
+        if not isinstance(node, SinglefileData):
+            raise click.BadArgumentUsage('Missing argument \'RELATIVE_PATH\'.')
+
+        relative_path = node.filename
```

Comment:
Remind me a bit what is the default filename if using the filelike for the `SinglefileData`?

Reply:
That is handled here https://github.com/aiidateam/aiida-core/blob/db0c25488526cb855bb04b095ee28355d2e64fbb/aiida/orm/nodes/data/singlefile.py#L93 so it is set to `file.txt` in that case.